alerts/alerts.libsonnet

```diff
@@ -1,6 +1,9 @@
-(import 'absent_alerts.libsonnet') +
 (import 'apps_alerts.libsonnet') +
 (import 'resource_alerts.libsonnet') +
 (import 'storage_alerts.libsonnet') +
 (import 'system_alerts.libsonnet') +
+(import 'system_alerts_apiserver.libsonnet') +
+(import 'system_alerts_kubelet.libsonnet') +
+(import 'system_alerts_scheduler.libsonnet') +
+(import 'system_alerts_controller_manager.libsonnet') +
```

Comment:
I wouldn't mind getting rid of the `system_alerts` prefix.

Reply:
Good point. I also normalized all the other naming schemes as alerts and rules were inconsistent. PTAL.